core/src/main/scala/org/apache/spark/storage/ShuffleBlockFetcherIterator.scala

```diff
@@ -102,7 +102,7 @@ final class ShuffleBlockFetcherIterator(
     checksumAlgorithm: String,
     shuffleMetrics: ShuffleReadMetricsReporter,
     doBatchFetch: Boolean,
-  clock: Clock = new SystemClock())
+    clock: Clock = new SystemClock())
```

Comment:
Although there's a bit of an issue with this indentation, perhaps there's no need to make changes.